### ISSUE TITLE
Allow opening invalid metadata descriptors

### ISSUE THREAD

**Issue by roll** [frictionlessdata/frictionless-py#1483]
# Overview

See https://app.slack.com/client/T0362UCGH70/C0362US1U3G/thread/C0362US1U3G-1680595643.062129?cdn_fallback=2

# Implementation

Metadata should not be validated by default (only if it's requested):

```
package = Package.from_descriptor(d) # allow invalid
package = Package.from_descriptor(d, validate=True)
```

We use the same approach for serialization:

```
package.to_descriptor(d) # allow invalid
package.to_descriptor(d, validate=True)
```

The downside - an invalid descriptor might cause unpredicted runtime behavior and fails (as it will break type safety), so it should be recommended always use `validate=True`

**Comment by roll** [comment]
Hi @davidgasquez,

It's being released as `v5.13` with `Resource/Package/etc.from_descriptor(allow_invalid=False)` flag available

**Comment by davidgasquez** [comment]
Awesome! Thanks a lot @roll! :raised_hands:
